external-crates/move/crates/move-unit-test/src/lib.rs

```diff
@@ -133,7 +131,7 @@ impl UnitTestingConfig {
             verbose: false,
             list: false,
             named_address_values: vec![],
-            rand_num_iters: 10,
+            rand_num_iters: Some(10),
```

Comment:
nit, should this be bound to a constant somewhere?

Reply:
darn it is...and I forgot to update it here.